Lib/zipfile/__init__.py

```diff
@@ -1361,6 +1362,7 @@ class ZipFile:
                    When using ZIP_STORED or ZIP_LZMA this keyword has no effect.
                    When using ZIP_DEFLATED integers 0 through 9 are accepted.
                    When using ZIP_BZIP2 integers 1 through 9 are accepted.
+                   When using ZIP_ZSTANDARD integers are accepted.
```

Comment:
```suggestion
                   When using ZIP_ZSTANDARD integers are accepted. The minimum and maximum values can be access via the CompressionParameter enum in compression.zstd.
```

Reply:
I agree that the documentation of the compression level requires some nuance (this is why I didn't initially specify a range). However, I do think there's value in having at least some common compression levels detailed for readers that only need the basics.

What about:
> When using ZIP_ZSTANDARD integers -7 though 22 are common, see the CompressionParameter enum in compression.zstd for details.

Reply:
My concern is that the values/their meaning are not stable, only their ordering. If we hard code them and a future version changes things, that might be an issue for users.

Reply:
Thinking about it more, I think I am fine with your proposed alternative wording.